fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchQuantityTest.java

```diff
@@ -170,163 +187,195 @@ public void testSearchQuantity_Quantity_NoDisplayUnit() throws Exception {
+
+        // In this test, the comparator should be from the DB
         // With implicit ranges, 3 (+/-0.5) actually might be > 3
-//      assertSearchDoesntReturnSavedResource("Quantity-greaterThan", "3||gt");
-        
+        // For more details, please see Quantity search does not consider the quantity comparator field #493 https://github.com/IBM/FHIR/issues/493
+        //assertSearchDoesntReturnSavedResource("Quantity-greaterThan", "3||gt");
+        assertSearchReturnsSavedResource("Quantity-greaterThan", "3||gt");
+
         assertSearchDoesntReturnSavedResource("Quantity-greaterThan", "lt2||gt"); // > 3 is not < 2
-        assertSearchReturnsSavedResource("Quantity-greaterThan", "gt2||gt");      // > 3 may be > 2
-        assertSearchReturnsSavedResource("Quantity-greaterThan", "lt4||gt");      // > 3 may be < 4 
-//        assertSearchReturnsSavedResource("Quantity-greaterThan", "gt4||gt");      // > 3 may be > 4
+        assertSearchReturnsSavedResource("Quantity-greaterThan", "gt2||gt"); // > 3 may be > 2
+        assertSearchReturnsSavedResource("Quantity-greaterThan", "lt4||gt"); // > 3 may be < 4
+        // For more details, please see Quantity search does not consider the quantity comparator field #493 https://github.com/IBM/FHIR/issues/493
+        // assertSearchReturnsSavedResource("Quantity-greaterThan", "gt4||gt");      // > 3 may be > 4
     }
-    
+
     @Test
     public void testSearchQuantity_Quantity_LessThanOrEqual() throws Exception {
-//        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "2||lte");
+        assertSearchDoesntReturnSavedResource("Quantity-lessThanOrEqual", "2||lte");
         assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "3||lte");
         assertSearchDoesntReturnSavedResource("Quantity-lessThanOrEqual", "4||lte");
-        
-//        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "lt2||lte");      // <= 3 may be < 2
-        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "gt2||lte");      // <= 3 may be > 2
-        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "lt4||lte");      // <= 3 may be < 4 
+
+        assertSearchDoesntReturnSavedResource("Quantity-lessThanOrEqual", "lt2||lte");      // <= 3 may be < 2
+        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "gt2||lte"); // <= 3 may be > 2
+        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "lt4||lte"); // <= 3 may be < 4 
         assertSearchDoesntReturnSavedResource("Quantity-lessThanOrEqual", "gt4||lte"); // <= 3 is not > 4
-        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "le3||lte");      // <= 3 is <= 3
-        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "ge3||lte");      // <= 3 may be >= 3
+        
+        // As we have added implict ranges to the prefix processing, we need to add precision 
+        // >= 3 may be <= 3 uses precision and bounding for the range. 
```

Comment:
I suggest removing this.  At least the first line and probably the second as well.